libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -398,11 +400,13 @@ static DNNReturnType add_conv_layer(TFModel *tf_model, TF_Operation *transpose_o
     memcpy(TF_TensorData(tensor), params->biases, params->output_num * sizeof(float));
     TF_SetAttrTensor(op_desc, "value", tensor, tf_model->status);
     if (TF_GetCode(tf_model->status) != TF_OK){
+        TF_DeleteTensor(tensor);
```

Comment:
op_desc  is also needed?

Reply:
I did not find any c_api like TF_DeleteOperation. And checked the test_c_api code of github tensorflow repo, it looks no need to free TF_NewOperation(..) returned mem.

Reply:
ok